testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -180,10 +182,7 @@
 
 When(/^I check Cobbler buildiso ISO "([^"]*)" with xorriso$/) do |name|
   tmp_dir = '/var/cache/cobbler'
-  out, code = get_target('server').run("cat >#{tmp_dir}/test_image <<-EOF
-BIOS
-UEFI
-EOF")
+  out, code = get_target('server').run("cat >#{tmp_dir}/test_image <<-EOF BIOS UEFI EOF")
```

Comment:
I am curious if this will work, especially because all words should be added to a new line.

Reply:
Indeed this smells very bad.I will revert it.